frontend/src/components/ViewTrips/create-new-trip.js

```diff
@@ -96,26 +96,36 @@ function formatTripData(rawTripObj) {
  * Promise containing a reference to the newly created document.
  *
  * @param {firebase.firestore.Firestore} db The Firestore database instance.
+ * @param {?string} tripId Document id associated with trip to overwrite.
```

Comment:
What does ?string do?

Reply:
It means that it is a nullable type: https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System.